File: FizzBuzz/FizzBuzz.py

fizz = lambda x : "Fizz" if (x%3==0) else ""
buzz = lambda x : "Buzz" if (x%5==0) else ""

def fizzbuzz(x):
    isFizzBuzz = fizz(x) + buzz(x)
    if isFizzBuzz!="":
        print(isFizzBuzz)
    else:
        print(x)

for i in range(1,101):
    fizzbuzz(i)

File: FizzBuzz/FizzBuzz.R


fizzbuzz <- function(n=100){
  sapply(1:n,function(i){
    fizz <- ifelse(i%%3==0,"Fizz","")
    buzz <- ifelse(i%%5==0,"Buzz","")
    fizzbuzz <- paste0(fizz,buzz)
    return(ifelse(fizzbuzz=="",i,fizzbuzz))
  })
}
